File: application/src/main/java/com/example/multimodule/application/MyController.java
package com.example.multimodule.application;

import com.example.multimodule.service.MyService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class MyController {
    private final MyService myService;

    public MyController(MyService myService) {
        this.myService = myService;
    }

    @GetMapping("/")
    public String home() {
        return myService.message();
    }
}

File: application/src/main/kotlin/com/example/multimodule/application/MyController.kt
package com.example.multimodule.application

import com.example.multimodule.service.MyService
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class MyController(
    private val myService: MyService,
) {
    @GetMapping("/")
    fun home(): String = myService.message()
}
